database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,13 +15,20 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
-            $table->string('name');
+            $table->bigIncrements('id');
+            $table->string('first_name')->nullable();
+            $table->string('last_name')->nullable();
+            $table->string('nick_name')->nullable();
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->date('date_of_birth')->nullable();
+            $table->text('about')->nullable();
+            $table->unsignedBigInteger('image_id')->nullable();
             $table->rememberToken();
-            $table->timestamps();
+            $table->tinyInteger('status')->nullable()->default(0);
+            $table->timestamp('created_at')->nullable()->useCurrent();
+            $table->timestamp('updated_at')->nullable()->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
```

Comment:
Why this instead of `$table->timestamps();` ?

Reply:
I tried using timestamps(), but when i try to add data, it is not automatically filled (the created_at and updated_at column). I will figure out when i am at this task again.. just put it in my kanban..